docs/6.Events.md

```diff
@@ -73,27 +73,27 @@ the `bootstrap` and/or `finish` event to do just this.
 public function attach(EventManagerInterface $events)
 {
     $this->listeners[] = $events->attach(
-        WorkerEvent::EVENT_BOOTSTRAP,
+        AbstractWorkerEvent::EVENT_BOOTSTRAP,
```

Comment:
Coudn't those constants be part of an "WorkerEventInterface" interface instead from an abstract class?